patches/tModLoader/Terraria/Localization/Content/fr-FR/tModLoader.json

```diff
@@ -3,22 +3,22 @@
 		// Main Menu
 		"PatreonSupport": "Considerez de nous supporter financièrement: ",
 		"MenuModBrowser": "Navigateur",
-		// "MenuDownloadMods": "Download Mods",
-		// "MenuDownloadModsDescription": "The in-game Mod Browser to find, download, and update your Mods.",
+		// "MenuDownloadMods": "Télécharger des Mods",
+		// "MenuDownloadModsDescription": "Le Bavigateur de Mods dans le jeu pour trouver, télécharger, et mettre à jour votre Mods.",
 		"MenuMods": "Mods",
-		// "MenuManageMods": "Manage Mods",
-		// "MenuManageModsDescription": "Manage which Mods are active, delete Mods, and configure your Modpacks.",
-		"MenuModSources": "Mods source",
-		// "MenuDevelopMods": "Develop Mods",
-		// "MenuDevelopModsDescription": "Develop and publish Mods for others to use, using your Mod Source code",
+		// "MenuManageMods": "Gérer les Mods",
+		// "MenuManageModsDescription": "Gérez quels Mods sont actifs, supprimez les mods, et configurez votre Modpacks.",
+		"MenuModSources": "Sources de Mods",
+		// "MenuDevelopMods": "Développer des Mods",
+		// "MenuDevelopModsDescription": "Développer et publier des Mods pour que d’autres puissent les utiliser, en utilisant votre Code Source Mod",
 		"MenuEnableDeveloperMode": "Activer le mode Développeur",
-		// "WelcomeMessageBeta": "Welcome to tModLoader 1.4 Alpha.\n\nYou have successfully installed an unstable tModLoader that may contain game-breaking bugs. \n\nUse the Workshop menu to access tModLoader Mod related functions.\n\nJoin the tModLoader Discord to learn more about which testing mods and/or tools are available for 1.4 tModLoader as part of its 1.4 overhaul.",
-		// "FirstLaunchWelcomeMessage": "Welcome to tModLoader.\n\nYou have successfully installed tModLoader. Use the Mod Browser menu to find mods you wish to download. Use the Mods menu to change which mods are currently active. After changing mods, make sure you click the Reload Mods button to reload them.",
-		// "WhatsNewMessage": "The following are basic change notes from the last time you launched the Alpha. You may find that mods that relate to these changes need to be rebuilt by the author. Visit GitHub to see more details:\n",
-		// "ViewOnGitHub": "View on GitHub",
-		// "tModLoaderSettings": "tModLoader Settings",
-		// "ModMenuSwap": "Switch Menu Theme",
-		// "AudioNotSupported": "No audio hardware found. Disabling all audio.",
+		// "WelcomeMessageBeta": "Bienvenue sur tModLoader 1.4 Alpha.\n\nVous avez installé avec succès une version instable de tModLoader qui peut contenir des bogues de rupture de jeu. \n\nUtilisez le menu Atelier pour accéder aux fonctions liées à tModLoader Mod.\n\nRejoignez le Discord tModLoader pour en savoir plus sur les mods et/ou outils de test disponibles pour tModLoader 1.4 dans le cadre de sa refonte 1.4.",
```

Comment:
Honestly, no one use "bogues" and nearly everyone talks about "bugs" in French, even if that's not what dictionaries say.
I'd understand to leave it as "bogue" though